backend/inspirehep/disambiguation/tasks.py

```diff
@@ -11,11 +11,11 @@
 
 LOGGER = structlog.getLogger()
 
-disambiguation_assigned_clusters_total = Counter(
-    "disambiguation_assigned_clusters_total", "Assigned clusters", ["num_authors"]
+disambiguation_assigned_clusters = Counter(
+    "disambiguation_assigned_clusters", "Assigned clusters", ["num_authors"]
```

Comment:
why the renaming?

Reply:
Prometheus adds `_total` to names of all counters automatically so setting this manually was unnecessary and wrong at the first place.